nose_gevented_multiprocess/nose_gevented_multiprocess.py

```diff
@@ -534,14 +534,22 @@ def run_clients(number_of_clients, server_port, stop_event):
         if waited is stop_event:
             # Clean up by terminating the clients
             for client in clients:
-                client.wait(timeout=1)
-                log.info("Terminating client %s" % client)
+                log.info("Killing client %s" % client)
                 try:
-                    client.terminate()
+                    client.kill()
```

Comment:
Do we want to try to allow the subprocesses to clean up gracefully before sending a SIGKILL? As is, this more or less guarantees that we're going to leak databases/temp files. If we send a SIGINT

Reply:
Yep, had pushed an intermediate version of that commit. Meant to wait and terminate and then wait and kill.